fastlane/Fastfile

```diff
@@ -941,14 +957,14 @@ inal copy, and try again.")
 
     # Add all development certificates to the provisioning profiles (just in case – this is an easy step to miss)
     add_development_certificates_to_provisioning_profiles(
-      team_id: TEAM_ID,
-      app_identifier: MAIN_BUNDLE_IDENTIFIERS
+      team_id: TEAM_ID_PRODUCTION,
+      app_identifier: PRODUCTION_BUNDLE_IDENTIFIERS
     )
 
     # Add all devices to the provisioning profiles
     add_all_devices_to_provisioning_profiles(
-      team_id: TEAM_ID,
-      app_identifier: MAIN_BUNDLE_IDENTIFIERS
+      team_id: TEAM_ID_PRODUCTION,
+      app_identifier: PRODUCTION_BUNDLE_IDENTIFIERS
     )
   end
```

Comment:
These last calls come from `register_new_device`, a lane that adds devices to the ASC account. We've long since improved this workflow via internal tooling. @Automattic/pocket-casts-ios is it okay to remove `register_new_device`?

Reply:
This obsolete lane also has the big drawback that any device registered using it would not be claimed and associated with the Matticspace user the device belongs to (and thus would be removed like any other unclaimed device when we do our yearly cleanup too). So 💯  to remove that lane.

Reply:
Addressed in https://github.com/Automattic/pocket-casts-ios/pull/1758